File: SkaiLab.Invoice/ClientApp/src/app/models/country.ts
export class Country{
    id:number;
    alpha2Code:string;
    alpha3Code:string;
    name:string;
    nationality:string;
    constructor(){
        this.id=0;
        this.alpha2Code="";
        this.alpha3Code="";
        this.name="";
        this.nationality="";
    }
}

File: SkaiLab.Invoice.Models/Country.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SkaiLab.Invoice.Models
{
    public class Country
    {
        public int Id { get; set; }
        public string Alpha2Code { get; set; }
        public string Alpha3Code { get; set; }
        public string Name { get; set; }
        public string Nationality { get; set; }

    }
}
